jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml

```diff
@@ -1,19 +1,16 @@
-# Common publishers shared by all e2e jobs.
-- publisher:
-    name: e2e-publishers
-    publishers:
-        - claim-build
-        - junit-publisher
-        - log-parser
-        - email-ext:
-            recipients: '{recipients}'
-        - gcs-uploader
+# This file defines e2e jobs against Kuberentes HEAD and GCI HEAD, from master
+# and the latest three release branches, and the GCI milestones that they pin
+# to, e.g., GCI milestone for Kubernetes `release-1.2`.
```

Comment:
The example looked incomplete to me. I was expecting the example will be for the map:
   e.g., GCI milestone for Kubernetes `release-1.2` is 52.
But I am probably reading it incorrectly.